skills/plane-backlog/scripts/plane_create_issue.py

```
@@ -0,0 +1,500 @@
+    }
+    
+    tiptap_doc, html_desc, plain_desc = markdown_to_tiptap_and_html(description)
+    payload = {
+        "name": title,
+        "description": tiptap_doc,
+        "description_html": html_desc,
+        "description_stripped": plain_desc
+    }
+
+    try:
+        req = urllib.request.Request(url, data=json.dumps(payload).encode("utf-8"), headers=headers, method="POST")
```

Ruff (0.16.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)